src/Stratis.Features.SQLiteWalletRepository.Tests/WalletRepositoryTests.cs

```diff
@@ -173,11 +174,15 @@ public class WalletRepositoryTests
         public WalletRepositoryTests(bool dbPerWallet = true)
         {
             this.dbPerWallet = dbPerWallet;
-            this.network = KnownNetworks.StratisTest;
+            //this.network = KnownNetworks.StratisTest;
+            this.network = KnownNetworks.StratisMain;
 
-            // Configure this to point to your "StratisTest" root folder and wallet.
-            this.walletNames = new[] { "test2", "test" };
-            this.dataDir = @"E:\RunNodes\SideChains\Data\MainchainUser";
+
+            //Configure this to point to your "StratisTest" root folder and wallet.
+            //this.walletNames = new[] { "test2", "test" };
+            //this.dataDir = @"E:\RunNodes\SideChains\Data\MainchainUser";
+            this.walletNames = new[] { "wallettABA" };
+            this.dataDir = @"C:\Dev\InternalTestnet_Data\Runtime\NodeMDA";
```

Comment:
This doesn't look right ;) Its a local folder?

Reply:
I believe that test is being skipped and there is a comment that it requires manual setup. It can be improved later but was useful for testing during development.